lnwallet/channel.go

```diff
@@ -5133,7 +5143,8 @@ func newOutgoingHtlcResolution(signer input.Signer,
 // they can just sweep the output immediately with knowledge of the pre-image.
 //
 // TODO(roasbeef) consolidate code with above func
-func newIncomingHtlcResolution(signer input.Signer, localChanCfg *channeldb.ChannelConfig,
+func newIncomingHtlcResolution(signer input.Signer,
```

Comment:
Are these funcs also candidates to move into `commitment.go`, now that they are channel type-dependent?

Reply:
Since we now just pass the `chanType` through, I think keeping them here is okay.